tests/test_circuit.py

```diff
@@ -29,6 +29,7 @@ def test_all():
     )
 
 
+@skip_if_libsonata_0_1_16
```

Comment:
Neat. I wonder, however, if, starting from the release, we would want to require `libsonata>=0.1.16` and remove these tests altogether. 🤔